docs/contributing/wc-guidelines.md

```diff
@@ -188,6 +189,78 @@ export class MyComponent extends LitElement {
 If your component deals with two different sets of data that do not share the same state (for instance one data set could be `loaded` while the other is still `loading`), you should create separate `state` props and name them with a meaningful prefix followed by `State`, for instance:
 `instancesState`, `applicationState`, `memberListState`, etc.
 
+### Subcomponent states
+
+Parent components should avoid exposing their subcomponent states.
+For instance when dealing with a parent component which manages a list of items, with each item being a subcomponent:
+- the parent should expose a standard `state` prop,
+- data to be passed to subcomponents (items data) should be plain data,
+- plain data should be transformed into a `state` by the parent component within its `render` or `subrender` methods before passing it to its subcomponents.
+
+
+When dealing 
+For instance, subcomponents like `cc-article-card` are only used by `cc-article-list`. 
+These subcomponents can either be `loaded` or `loading` but their state is actually dictated by a single API call that loads all articles at once.
+
+In such cases:
+
+- the subcomponent (`cc-article-card`) should still expose a `state` API
+- the parent component (`cc-article-list`) should expose a global `state` prop. When `loaded`, the  APIshould be responsible for handling the state of all its subcomponents within its `render` method.
+
+A good example is [cc-article-card](https://github.com/CleverCloud/clever-components/blob/master/src/components/cc-article-card/cc-article-card.js) and [cc-article-list](https://github.com/CleverCloud/clever-components/blob/master/src/components/cc-article-list/cc-article-list.js).
```

Comment:
nitpick: we already talk about `cc-article-card` and `cc-article-list` above. Plus, we have an example below. I think it would be better if we had the link either above or below after the example, it would make more sense.

Reply:
Hmpf I was supposed to remove this, thanks, you're right indeed :+1: